bookmarks/services/parser.py

```diff
@@ -53,7 +118,7 @@ def extract_description(tag):
 bookmark_tag.addParseAction(extract_bookmark)
 
 
-def parse(html: str) -> [NetscapeBookmark]:
+def parse(html: str) -> List[NetscapeBookmark]:
```

Comment:
Out of curiousity, why use `List[...]` rather than `[...]`?

Reply:
To my understanding `[...]` is incorrect. The brackets are not type annotations like in C/C++ (`int vec[]` being an array of `int`s) but sort of a parenthesis for specifying generic types, like in a C++ template specifier (`template MyType<class T>`).